include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -786,6 +786,7 @@ struct OrtApi {
    * on the SessionOptions instance used to create the session.
    */
   ORT_API2_STATUS(SessionEndProfiling, _In_ OrtSession* sess, _Inout_ OrtAllocator* allocator, _Outptr_ char** out);
+  ORT_API2_STATUS(SessionGetProfilingStartTime, _In_ OrtSession* sess, _Outptr_ uint64_t* out);
```

Comment:
can session be passed as a const-ptr?

Reply:
Needs documentation about the unit of time returned. Can also rename this to SessionGetProfilingStartTimeNs (to indicate nanoseconds).

Reply:
I tried to pass the session as const-ptr but failed. I saw all of the return values here are either integers or characters and I failed to include other header file to introduce other class in that file.

I added some comments and change the function name to make it clearer. Thank you for the suggestion!